insight.go

```diff
@@ -76,6 +78,7 @@ type CarrierRecord struct {
 	NetworkType string `json:"network_type"`
 }
 
+// Peform a Number Insight request at a standard level of detail
```

Comment:
s/Peform/Perform/g

Reply:
Copy paste curse! There were three of them!